examples/frigate/frigate.ini

```diff
@@ -0,0 +1,56 @@
+[defaults]
+functions = frigate.py
+launch    = apprise-ntfy, store-jpeg
+
+status_publish = True
+
+
+
+# =====================
+# Frigate event to Ntfy
+# =====================
+
+# Format: JSON
+# Docs:   https://docs.frigate.video/integrations/mqtt/#frigateevents
+
+
+
+[config:apprise-ntfy]
+module   = apprise_single
+baseuri  = ntfys://ntfy/frigate
+filename_template = './var/media/{camera}-{label}.jpg'
+
+
+[frigate/events]
+filter  = frigate_events_filter()
+alldata = frigate_events()
+targets = apprise-ntfy
+title   = {title}
+format  = {format}
+click   = {click}
+
+
+
+# =====================
+# Frigate image to file
+# =====================
+
+# Format: JPEG
+# Docs:   https://docs.frigate.video/integrations/mqtt/#frigatecamera_nameobject_namesnapshot
+
+[config:store-jpeg]
+module         = file
+targets        = {
+    'cam-testdrive-goat':        ['./var/media/cam-testdrive-goat.jpg'],
+    'cam-testdrive-squirrel':    ['./var/media/cam-testdrive-squirrel.jpg'],
+    }
+
+# Configure `file` plugin to pass through payload 1:1.
+append_newline = False
+decode_utf8    = False
+overwrite      = True
+
+[frigate/cam-testdrive/goat/snapshot]
+targets = store-jpeg:cam-testdrive-goat
+[frigate/cam-testdrive/squirrel/snapshot]
+targets = store-jpeg:cam-testdrive-squirrel
```

Comment:
This section may be condensed, depending on your taste, see [Templated targets » Example 2](https://mqttwarn.readthedocs.io/en/latest/configure/topic.html#example-2).